parlai/crowdsourcing/tasks/turn_annotations_static/webapp/src/components/task_components.jsx

```diff
@@ -163,34 +195,57 @@ function SubtaskSubmitButton({ subtaskIndex, numSubtasks, onSubtaskSubmit }) {
   )
 }
 
-function ChatMessage({ text, agentIdx, annotationQuestion, annotationBuckets, turnIdx, doAnnotateMessage, askReason, onUserInputUpdate }) {
+function ChatMessage({ text, agentIdx, annotationQuestion, annotationBuckets, turnIdx, doAnnotateMessage, askReason, responseField, onUserInputUpdate }) {
   var extraElements = '';
+  var responseInputElement = '';
   if (doAnnotateMessage) {
-    extraElements = '';
-    extraElements = (<span key={'extra_' + turnIdx}><br /><br />
-      <span style={{ fontStyle: 'italic' }} >
-        <span dangerouslySetInnerHTML={{ __html: annotationQuestion }}></span>
-        <br />
-        <Checkboxes turnIdx={turnIdx} annotationBuckets={annotationBuckets} askReason={askReason} onUserInputUpdate={onUserInputUpdate} />
-      </span>
-    </span>)
+    if (annotationBuckets) { 
+      extraElements = (<span key={'extra_' + turnIdx}><br /><br />
+        <span style={{ fontStyle: 'italic' }} >
+          <span dangerouslySetInnerHTML={{ __html: annotationQuestion }}></span>
+          <br />
+          <Checkboxes turnIdx={turnIdx} annotationBuckets={annotationBuckets} askReason={askReason} onUserInputUpdate={onUserInputUpdate} />
+        </span>
+      </span>)
+    }
+    if (responseField) {
+      responseInputElement = (
+        <FormControl
+        type="text"
+        name="input_response"
+        id={"input_response_" + turnIdx}
+        style={{
+            fontSize: "14px",
+            resize: "none",
+            marginBottom: "40px"
+        }}
+        onChange={(e) => {onUserInputUpdate();}}
+        placeholder={"Please enter your response here"}
+        onPaste={(e) => {e.preventDefault(); alert("Please do not copy and paste and respond to each message as you would!")}}
```

Comment:
Nit: I'm not sure this is quite saying the right thing to a worker due to the chaining ands, though I'm not really sure what asking them to "Please do not ... respond to each message as you would!" is even implying. Perhaps "Please do not copy and paste. You must manually respond to each message." is better?